### ISSUE TITLE
Sub Users.

### ISSUE THREAD

**Issue by Name12MH** [Minehut/Meta#785]
### Feature request

**Checklist**
- [x] I have read and agree to the README (found on https://github.com/Minehut/Meta)
- [ x] This is not a bug report, support request or plugin request
- [x ] I have searched the Issue Tracker to make sure this is not a duplicate issue
- [ x] This issue does not contain any sensitive information 

**Platform**

- Both

**Describe the feature**

Hi!

I had an idea for a future update, if you could check this out and or respond that would be amazing!

So a lot of servers have this problem, or at least my server, where they want to give someone like an admin / developer / etc console, but doesn’t want them to be able to delete everything, download stuff, etc, so what I was thinking is:

You should add a feature where you can add an email to your console and set their login code they have to use to login BUT to make it better in your guys’s end I was thinking this, each plan gets a certain amount of emails they can add to that console.

FREE: 0
MH20: 1
MH35 2
MH75 4 
MH-UNLIMITED: 7

So basically what it would do is when you add the email, thar person can login with the email and the login key, and then you can manage the perms they have on the console, like you can toggle file manager access console command access credit spending access purchasing access and world / backup / etc access, and you can for example it would say:

JohnDoe@gmail.com | File Manager:
  Download files: FALSE
  Upload files: FALSE
  View files: TRUE
  Edit files: TRUE
  Create new file: FALSE

And you can toggle them on and off,

- Name

**How the feature is useful**

This feature would be very helpful to the minehut community, thank you for reading!

---
**Use 👍 and 👎 reactions to indicate you agree or disagree with this suggestion! Feel free to discuss the idea in the comments.**

**Comment by CoreyShupe** [comment]
#13
